corehq/pillows/synclog.py

```diff
@@ -30,7 +30,7 @@ def get_user_sync_history_pillow(
     change_feed = KafkaChangeFeed(
         topics=[topics.SYNCLOG_SQL], group_id=SYNCLOG_SQL_USER_SYNC_GROUP_ID,
         num_processes=num_processes, process_num=process_num)
-    checkpoint = KafkaPillowCheckpoint('synclog-user-sync', [topics.SYNCLOG_SQL])
+    checkpoint = KafkaPillowCheckpoint('synclog', [topics.SYNCLOG_SQL])
```

Comment:
why change this? In most cases we make this equal to `pillow_id`

Reply:
I see you were just copying from what was there before. I think better to change this to be `pillow_id`

Reply:
Reverted and fixed the test data instead